test/Main.purs

```diff
@@ -4,6 +4,9 @@ import Prelude
 
 import Effect (Effect)
 import Test.Data.Enum (testEnum)
+import Test.Data.GenericsRep (testGenericsRep)
 
 main :: Effect Unit
-main = testEnum
+main = do
+  testEnum
+  testGenericsRep
```

Comment:
What about testGenericEnum, since we’re testing generic implementations for Enum and BoundedEnum?

Reply:
Fixed